packages/hop-node/src/watchers/RelayWatcher.ts

```diff
@@ -0,0 +1,383 @@
+import { L2Bridge as L2BridgeContract } from '@hop-protocol/core/contracts/L2Bridge'
+import { NonceTooLowError, RelayerFeeTooLowError } from 'src/types/error'
+import { RelayableTransferRoots } from 'src/db/TransferRootsDb'
+import { Transfer, UnrelayedSentTransfer } from 'src/db/TransfersDb'
+import { isExecutionError } from 'src/utils/isExecutionError'
+import { promiseQueue } from 'src/utils/promiseQueue'
+import { providers } from 'ethers'
+import { relayTransactionBatchSize } from 'src/config'
+
+type Config = {
+  chainSlug: string
+  tokenSymbol: string
+  bridgeContract: L1BridgeContract | L2BridgeContract
+  dryMode?: boolean
+}
+
+type RelayWatchers = ArbitrumBridgeWatcher
+
+class RelayWatcher extends BaseWatcher {
+  siblingWatchers: { [chainId: string]: RelayWatcher }
+  relayWatcher: RelayWatchers
+
+  constructor (config: Config) {
+    super({
+      chainSlug: config.chainSlug,
+      tokenSymbol: config.tokenSymbol,
+      logColor: 'redBright',
+      bridgeContract: config.bridgeContract,
+      dryMode: config.dryMode
+    })
+    this.relayWatcher = new ArbitrumBridgeWatcher({
+      chainSlug: config.chainSlug,
+      tokenSymbol: this.tokenSymbol,
+      bridgeContract: config.bridgeContract,
+      dryMode: config.dryMode
+    })
+  }
+
+  async pollHandler () {
+    if (this.isL1) {
+      return
+    }
+
+    const promises: Array<Promise<any>> = []
+    promises.push(
+      this.checkTransferSentToL2FromDb(),
+      this.checkRelayableTransferRootsFromDb()
+    )
+    await Promise.all(promises)
+  }
+
+  async checkTransferSentToL2FromDb () {
+    const dbTransfers = await this.db.transfers.getUnrelayedSentTransfers(await this.getFilterRoute())
+    if (!dbTransfers.length) {
+      this.logger.debug('no unrelayed transfer db items to check')
+      return
+    }
+
+    this.logger.info(
+      `total unrelayed transfers db items: ${dbTransfers.length}`
+    )
+
+    const listSize = 100
+    const batchedDbTransfers = dbTransfers.slice(0, listSize)
+
+    this.logger.info(
+      `checking unrelayed transfers db items ${batchedDbTransfers.length} (out of ${dbTransfers.length})`
+    )
+
+    await promiseQueue(batchedDbTransfers, async (dbTransfer: Transfer, i: number) => {
+      const {
+        transferId
+      } = dbTransfer
+      const logger = this.logger.create({ id: transferId })
+      logger.debug(`processing item ${i + 1}/${batchedDbTransfers.length} start`)
+      logger.debug('checking db poll')
+
+      try {
+        logger.debug('checkTransferSentToL2 start')
+        await this.checkTransferSentToL2(transferId)
+      } catch (err: any) {
+        logger.error('checkTransferSentToL2 error:', err)
+      }
+
+      logger.debug(`processing item ${i + 1}/${batchedDbTransfers.length} complete`)
+      logger.debug('db poll completed')
+    }, { concurrency: relayTransactionBatchSize, timeoutMs: 10 * 60 * 1000 })
+
+    this.logger.debug('checkTransferSentFromDb completed')
```

Comment:
The log should say `checkTransferSentToL2FromDb` instead of `checkTransferSentFromDb`